commands/operator-sdk/cmd/new.go

```diff
@@ -161,6 +161,11 @@ func execCmd(stdout *os.File, cmd string, args ...string) {
 }
 
 func pullDep() {
+	depPath, err := exec.LookPath(dep)
+	if err != nil {
+		cmdError.ExitWithError(cmdError.ExitError, err)
```

Comment:
Have you tested out what error message this will return?

It is possible that the err is not informative enough. We might add 
`cmdError.ExitWithError(cmdError.ExitError, fmt.Errorf("dep is not installed", err))` if that's the case.